extensions/replication/utils/SetupReplication.js

```diff
@@ -100,8 +100,8 @@ class SetupReplication extends BackbeatTask {
      */
     constructor(params) {
         const { source, target, https, internalHttps, checkSanity,
-                retryTimeoutS, skipSourceBucketCreation, log } = params;
-        super({ timeoutS: retryTimeoutS });
+                repConfig, skipSourceBucketCreation, log } = params;
+        super(repConfig.queueProcessor.retry.scality);
```

Comment:
replication may target other backends (not just scality), and each "kind" (scality, aws, azure...) has its own retry params:
- seems weird to single-out the 'scality' params here
- due to these target-specific params, are the params not used deeper in the code, when doing the actual replication?